components/Onboard/OnboardForm.js

```diff
@@ -422,6 +433,39 @@ class OnboardForm extends Component {
             id: "description",
             value: this.state.description,
           })}
+          {/* Social Links */}
+          <div className={css(styles.inputLabel, styles.text)}>
+            Social Links
+          </div>
+          <div className={css(styles.formInputContainer)}>
+            <FormInput
+              id="linkedin"
+              onChange={this.handleFormChange}
+              containerStyle={[styles.formInput, styles.socialInput]}
+              value={this.state.linkedin}
+              icon={<FontAwesomeIcon icon={faLinkedin} />}
+              iconStyles={[styles.socialIcon, styles.linkedInIcon]}
+              placeholder="https://linkedin.com/in/username"
+            />
+            <FormInput
+              id="twitter"
+              onChange={this.handleFormChange}
+              containerStyle={[styles.formInput, styles.socialInput]}
+              value={this.state.twitter}
+              icon={<FontAwesomeIcon icon={faSquareXTwitter} />}
+              iconStyles={[styles.socialIcon, styles.xTwitterIcon]}
+              placeholder="https://twitter.com/username"
```

Comment:
Maybe x.com instead of twitter as placeholder?

Reply:
updated!